src/cls/_ZPM/PackageManager.cls

```diff
@@ -535,16 +537,19 @@ ClassMethod Repository(ByRef pCommandInfo) [ Internal ]
 	}
 }
 
-ClassMethod ShowModulesForRepository(pRepoName As %String) [ Private ]
+ClassMethod ShowModulesForRepository(pRepoName As %String, pShowRepo As %Boolean = 0) [ Private ]
 {
-	Set tQuery = "select Name,Version from %ZPM_PackageManager_Developer.Utils_GetModuleList(?)"
+	Set tQuery = "select Name,Version,Repo from %ZPM_PackageManager_Developer.Utils_GetModuleList(?)"
 	Set tRes = ##class(%SQL.Statement).%ExecDirect(,tQuery,pRepoName)
 	If (tRes.%SQLCODE < 0) {
 		Throw ##class(%Exception.SQL).CreateFromSQLCODE(tRes.%SQLCODE,tRes.%Message)
 	}
 	While tRes.%Next(.tSC) {
 		$$$ThrowOnError(tSC)
 		Write !,tRes.%Get("Name")," ",tRes.%Get("Version")
+		If (pShowRepo) {
+			Write ?40," ",tRes.%Get("Repo")
```

Comment:
why 40?
Name and version not positioned, maybe better to fix position for all values

Reply:
add column for version